esx_service/utils/vmdk_perf.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##
+## Implements the interface for side car based implementation
+## of a KV store for vmdks.
```

Comment:
so do you expect the stats to be kept in the sidecar? Which one ? (I guess it could be in metric_short_labels but please elaborate).  Please add comments somewhere about the format used, how many samples you plan to keep in the file, and which samples.